### ISSUE TITLE
Scaffold resource checkboxes should have clickable labels

### ISSUE THREAD

**Issue by robdekort** [statamic/cms#2488]
## Bug Description
These labels should be clickable for a11y and a little better ux.
<img width="986" alt="Screenshot 2020-09-21 at 20 18 20" src="https://user-images.githubusercontent.com/69107412/93805214-afc4a880-fc47-11ea-8602-d7b89c5b39a8.png">

## Environment
Statamic 3.0.10 Pro
Laravel 7.28.3
PHP 7.4.6
